pytensor/xtensor/shape.py

```diff
@@ -504,3 +505,63 @@ def expand_dims(x, dim=None, create_index_for_new_dim=None, axis=None, **dim_kwa
         x = Transpose(dims=tuple(target_dims))(x)
 
     return x
+
+
+class XBroadcast(XOp):
```

Comment:
I just realized all the other ops are subclasses of XOp but don't have the leading `X` themselves.

Reply:
The MetaOps have, but you're right. Besides those only this and Dot have. I'll remove for consintency (and also Dot)